src/tests/js_hash/ion_hash_complex.ts

```diff
@@ -0,0 +1,102 @@
+import * as ion from 'ion-js';
+import * as ionHash from 'ion-hash-js';
+import * as crypto from "crypto";
+
+// This function will print a strict ready to copy past into rust
```

Comment:
a wat?

Reply:
This is the code I used in order to generate the computed hash for a test. I left it in here for reproducibility.